src/config.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const path = require('path')
+
+const config = {
+  version: 'v0.4.1', // ipfs dist version
+  defaultExecPath: path.join(__dirname, '..', 'vendor'), // default path for the downloaded exec
+  gracePeriod: 7500 // amount of ms to wait before sigkill
+}
+
+config.baseUrl = 'https://dist.ipfs.io/go-ipfs/' +
+  config.version +
+  '/go-ipfs_' +
+  config.version + '_'
```

Comment:
Yeah, true! I was also thinking on using template strings, something like:
```
const version = config.version
config.baseUrl = `https://dist.ipfs.io/go-ipfs/${version}/go-ipfs_${version}_`
```
Do you prefer this or the `Array.join`?